packages/client/hmi-client/src/components/models/tera-model-diagram.vue

```diff
@@ -1,87 +1,77 @@
 <template>
 	<main>
-		<TeraResizablePanel v-if="!nodePreview">
-			<div ref="splitterContainer" class="splitter-container">
-				<Splitter :gutterSize="5" :layout="layout">
-					<SplitterPanel
-						class="tera-split-panel"
-						:size="equationPanelSize"
-						:minSize="equationPanelMinSize"
-						:maxSize="equationPanelMaxSize"
-					>
-						<section class="graph-element">
-							<Toolbar>
-								<template #start>
+		<Accordion v-if="!nodePreview" multiple :activeIndex="[0, 1]" class="accordion-group">
+			<AccordionTab header="Model diagram">
+				<TeraResizablePanel class="diagram-container">
```

Comment:
Do you need those `<TeraResizablePanel>`?

Reply:
Yes, we still want to be able to change the size of the diagram and equations containers